Add docs for DisableRuntimeMarshalling (#36497)
dotnet/docs: docs/core/project-sdk/msbuild-props.md

</PropertyGroup>
```

### DisableRuntimeMarshalling

> Build-in runtime marshalling can be disabled by applying the [`System.Runtime.CompilerServices.DisableRuntimeMarshallingAttribute`](xref:System.Runtime.CompilerServices.DisableRuntimeMarshallingAttribute) attribute to an assembly. Guidance for disabling this feature can be found [here](../../standard/native-interop/disabled-marshalling.md).

Reviewer: nit
```suggestion
Built-in runtime marshalling can be disabled by applying the [`System.Runtime.CompilerServices.DisableRuntimeMarshallingAttribute`](xref:System.Runtime.CompilerServices.DisableRuntimeMarshallingAttribute) attribute to an assembly. Guidance for disabling this feature can be found [here](../../standard/native-interop/disabled-marshalling.md).
```

> Built-in runtime marshalling can be disabled by applying the [`System.Runtime.CompilerServices.DisableRuntimeMarshallingAttribute`](xref:System.Runtime.CompilerServices.DisableRuntimeMarshallingAttribute) attribute to an assembly. Guidance for disabling this feature can be found [here](../../standard/native-interop/disabled-marshalling.md).

Reviewer 2: ```suggestion
The `DisableRuntimeMarshalling` property enables you to specify that you would like to disable runtime marshalling support for your project. If this property is set to `true`, then the <xref:System.Runtime.CompilerServices.DisableRuntimeMarshallingAttribute> will be added to the assembly and any P/Invokes or delegate-based interop will follow the rules for [disabled runtime marshalling](../../standard/native-interop/disabled-marshalling).
```

> The `DisableRuntimeMarshalling` property enables you to specify that you would like to disable runtime marshalling support for your project. If this property is set to `true`, then the <xref:System.Runtime.CompilerServices.DisableRuntimeMarshallingAttribute> will be added to the assembly and any P/Invokes or delegate-based interop will follow the rules for [disabled runtime marshalling](../../standard/native-interop/disabled-marshalling).

Reviewer 2: ```suggestion
The `DisableRuntimeMarshalling` property enables you to specify that you would like to disable runtime marshalling support for your project. If this property is set to `true`, then the <xref:System.Runtime.CompilerServices.DisableRuntimeMarshallingAttribute> will be added to the assembly and any P/Invokes or delegate-based interop will follow the rules for [disabled runtime marshalling](../../standard/native-interop/disabled-marshalling.md).
```

> The `DisableRuntimeMarshalling` property enables you to specify that you would like to disable runtime marshalling support for your project. If this property is set to `true`, then the <xref:System.Runtime.CompilerServices.DisableRuntimeMarshallingAttribute> will be added to the assembly and any P/Invokes or delegate-based interop will follow the rules for [disabled runtime marshalling](../../standard/native-interop/disabled-marshalling.md).

Reviewer 3: ```suggestion
The `DisableRuntimeMarshalling` property enables you to specify that you would like to disable runtime marshalling support for your project. If this property is set to `true`, then the <xref:System.Runtime.CompilerServices.DisableRuntimeMarshallingAttribute> is added to the assembly and any P/Invokes or delegate-based interop will follow the rules for [disabled runtime marshalling](../../standard/native-interop/disabled-marshalling.md).
```

</PropertyGroup>
```

### DisableRuntimeMarshalling

Build-in runtime marshalling can be disabled by applying the [`System.Runtime.CompilerServices.DisableRuntimeMarshallingAttribute`](xref:System.Runtime.CompilerServices.DisableRuntimeMarshallingAttribute) attribute to an assembly. Guidance for disabling this feature can be found [here](../../standard/native-interop/disabled-marshalling.md).

> 

Reviewer 2: ```suggestion
```

</PropertyGroup>
```

### DisableRuntimeMarshalling

> TODO

Reviewer 2: ```suggestion
Build-in runtime marshalling can be disabled by applying the [`System.Runtime.CompilerServices.DisableRuntimeMarshallingAttribute`](xref:System.Runtime.CompilerServices.DisableRuntimeMarshallingAttribute) attribute to an assembly. Guidance for disabling this feature can be found [here](../../standard/native-interop/disabled-marshalling.md).
```

> TODO

Reviewer 2: I like this more than what I wrong. Will replace with yours.

> ### GenerateDisableRuntimeMarshallingAttribute
> 
> The `GenerateDisableRuntimeMarshallingAttribute` property is closely related to the [DisableRuntimeMarshalling](#disableruntimemarshalling) property. 
> 
> TODO

Reviewer 2: ```suggestion
```
